external/model-preparation-algorithm/mpa_tasks/apis/classification/task.py

```diff
@@ -300,6 +300,16 @@ def _init_model_cfg(self):
         else:
             cfg_path = os.path.join(base_dir, 'model.py')
         cfg = MPAConfig.fromfile(cfg_path)
+
+        # To initialize different HP for multi-label classification / Support HP change via CLI & UI
+        if self._multilabel:
+            template = MPAConfig.fromfile(self.template_file_path)
+            template_params = template.hyper_parameters.parameter_overrides.learning_parameters
+            if template_params.learning_rate.default_value != self._hyperparams.learning_parameters.learning_rate:
```

Comment:
This hack won't update the value on UI unfortunately: user will see 0.007 by default instead of actual 0.025 for some models.